docs/demo.mdx

```diff
@@ -0,0 +1,370 @@
+#              Lorem - Elementum - Consequitus
+```
+
+**Bash**
+
+```bash
+npm install "dummy_package" --save-dummy
+```
+
+**JavaScript**
+
+```javascript
+import { DummyButton } from 'dummy-react-library';
+
+function DummyExample() {
+  const [dummyState, setDummyState] = React.useState(false);
+
+  return (
+    <DummyContainer>
+      <DummyHeading>This is a dummy example</DummyHeading>
+      <DummyButton onClick={() => setDummyState(!dummyState)} isActive={dummyState}>
+        Click me
+      </DummyButton>
+      {dummyState && <DummyText>Button is clicked</DummyText>}
+    </DummyContainer>
+  );
+}
+
+<DummyExample />;
+```
+
+**Java**
+
+```java
+public class DummyClass {
+    public static void main(String[] args) {
+        int dummyVariable = 5;
+        for (int i = 0; i < 10; i++) {
+            dummyVariable += i;
+            System.out.println("The dummy value is " + dummyVariable);
+        }
+    }
+}
+```
+
+**shell**
+
+```sh
+# This is a simple dummy script
+
+# Define a dummy variable
+dummy_variable="Hello, World!"
+
+# Print the dummy variable
+echo $dummy_variable
+
+# Run a dummy command
+dummy_command arg1 arg2
+
+# Check the exit status of the last command
+if [ $? -eq 0 ]; then
+    echo "dummy_command succeeded"
+else
+    echo "dummy_command failed"
+fi
+```
+
+## Syntax Highlighting Section:
+
+**JSON**
+
+```json
+{
+  "items": {
+    "object1": {
+      "property1": "dummy_type",
+      "property2": "dummy_device",
+      "property3": ["dummy_path/to/dummy_file.dummy"]
+    }
+  }
+}
+```
+
+**YAML**
+
+```yaml
+actions:
+  - dummy install example
+    - push_dummytext: "*"
+
+  - dummy install example --save global
+  # Lorem ipsum dolor sit amet, consectetur SampleCo is using ($DUMMY/lorem-text)
+```
+
+**JSX with title**
+
+```jsx title="/src/dummycomponents/HelloDummyFunction.js"
+title = '/src/dummycomponents/HelloCDummyFunctions.js';
+function dummyFunction(props) {
+  let dummyVariable = 0;
+  return <h1>Hello, {props.dummyVariable}</h1>;
+}
+```
+
+**JAVA**
+
+```java
+ @Rule // (2)
+// highlight-next-line
+    public class DummyTestRule<DummyActivity> {
+    public DummyTestRule<DummyActivity> mDummyRule = new DummyTestRule<>(DummyActivity.class, false, false);
+
+    @Test
+    public void runDummyTests() {
+        DummyConfig dummyConfig = new DummyConfig();
+        dummyConfig.dummyPolicyConfig.dummyTimeoutSec = 90;
+    }
+}
+```
+
+**JavaScript**
+
+```javascript
+module.exports = {
+  things: {
+    'dummy.debug': {
+      type: 'dummy.apk',
+      // highlight-start
+      binaryPath: 'dummy/location/dummy.file',
+      build: 'dummy_command'
+      // highlight-end
+    }
+  }
+};
+```
+
+**`Plain Text` code span with line highlighting:**
+
+```plain text
+// highlight-next-line
+Pixel_3a_API_30_x86
+Pixel_API_30
+```
+
+## Image Sample Section:
+
+![](https://via.placeholder.com/728x450.png?text=Isquix+ConsecteturAdipiscing.ips+Siquis+lorem)
+
+## FlavorizedCodeBlock Section:
```

Comment:
FlavorizedCodeBlock is just a wrapper. It is not interesting. We need only the built in \`\`\` codeblocks and that's it.

Reply:
That concerns also `CodeBlock` component – you can use plain markdown, the three ticks syntax.

Reply:
I removed that "Flavorized codeblocks" section and created a ```diff and removed the wrapper - also added it to the codeblocks section